bundle.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: atlantis
+spec:
+  serviceName: atlantis
+  replicas: 1
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      partition: 0
+  selector:
+    matchLabels:
+      app: atlantis
+  template:
+    metadata:
+      labels:
+        app: atlantis
+    spec:
+      securityContext:
+        fsGroup: 1000 # Atlantis group (1000) read/write access to volumes.
+      containers:
+      - name: atlantis
+        image: runatlantis/atlantis:v0.9.0 
```

Comment:
Can you follow the same pattern as the Kubernetes manifest deployment instructions and use
```
        image: runatlantis/atlantis:v<VERSION> # 1. Replace <VERSION> with the most recent release.
```
Then include this in your patch instructions? Or how do kustomize templates usually work? If that's not possible then add updating this file to the release instructions in CONTRIBUTING.md

Reply:
We can - however from most examples I see, a lot of these bundles will be generated upon release and include the latest version in the bundle/kustomization file. It gives the user the a way to run the remote resource without any _manifest_ issues. Sure, Atlantis won't run if they don't patch the environment variables - however that's the application failing here, rather than the actual Kubernetes manifest.

Let me know your thoughts on this.

Reply:
That makes sense. Can you add a bullet to our `CONTRIBUTING.md` file under "releases" to say update the `kustomize/bundle.yaml` file. You'll see we already say to update `main.go`.